src/client/kv_client.rs

```diff
@@ -27,49 +31,94 @@ impl KvResponse {
     pub fn cas(&self) -> u64 {
         self.cas
     }
+
+    pub fn id(&self) -> String {
```

Comment:
since this is all internal, why use both id and key? (maybe just settle on one?)